packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/lib_router/router.dart

```diff
@@ -31,7 +35,9 @@ part 'router.g.dart';
 part 'routes/onboarding_routes.dart';
 part 'routes/profile_routes.dart';
 part 'routes/routes.dart';
+{{#enable_feature_deeplinks}}
```

Comment:
The show case routes includes not only the deep-links show case but the counter and widget toolkit as well. By excluding this file the app won't compile when the counter or the widget toolkit features are enabled